.github/workflows/unit-test.yml

```diff
@@ -122,7 +122,9 @@ jobs:
         uses: golangci/golangci-lint-action@v3.6.0
         with:
           version: latest
-          args: --timeout=10m --whole-files
+          args: --timeout=10m --whole-files -v
+          skip-pkg-cache: true
```

Comment:
Doesn't this disable caching?

We might want to instead try running this job with the official instructions which claim that caching is enabled by default:

```
jobs:
  golangci:
    name: lint
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-go@v4
        with:
          go-version: '1.20'
          cache: false
      - name: golangci-lint
        uses: golangci/golangci-lint-action@v3
...
```

Reply:
I've added `skip-pkg-cache` to address a similar issue to this [one](https://github.com/golangci/golangci-lint-action/issues/23).
I'll try your suggested setup as well, however, I'm still experimenting so I'll make it a draft pr.